src/s3/commands/addBucket.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getLogger } from '../../shared/logger'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { Commands } from '../../shared/vscode/commands'
+import { Window } from '../../shared/vscode/window'
+import { S3Node } from '../explorer/s3Nodes'
+import { showErrorWithLogs } from '../../shared/utilities/messages'
+import { validateBucketName } from '../util'
+
+export async function addBucketCommand(
```

Comment:
a short docstring helps others notice easily the difference vs "create bucket"
```suggestion
/**
 * Shows an explicitly-specified bucket in the AWS treeview.
 * 
 * To create a new bucket resource, see `createBucketCommand()`.
 */
export async function addBucketCommand(
```